# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../isdn4k-utils/isdn.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./isdnlog/isdnlog/isdnlog.h.orig    Fri Dec 31 05:30:02 1999
+++ ./isdnlog/isdnlog/isdnlog.h Fri Jun 29 11:38:15 2001
@@ -284,6 +284,14 @@
 #include <rate.h>
 #include "socket.h"
 
+/* Ugly hack, but the isdn header now defines it in kernel space. */
+#ifndef ISDN_MAX_CHANNELS
+#  define ISDN_MAX_CHANNELS 64
+#endif
+#ifndef ISDN_MAX_DRIVERS
+#  define ISDN_MAX_DRIVERS 32
+#endif
+
 /****************************************************************************/
 
 #define MAX_CALLS_IN_QUEUE    100
--- ./xisdnload/xisdnload.c.orig	Wed Jun 23 06:01:36 1999
+++ ./xisdnload/xisdnload.c	Fri Jun 29 11:13:42 2001
@@ -60,6 +60,14 @@
 
 #include "xisdnload.bit"
 
+/* Ugly hack, but the isdn header now defines it in kernel space. */
+#ifndef ISDN_MAX_CHANNELS
+#  define ISDN_MAX_CHANNELS 64
+#endif
+#ifndef ISDN_MAX_DRIVERS
+#  define ISDN_MAX_DRIVERS 32
+#endif
+
 char *ProgramName;
 
 static void quit();
--- ./imontty/imontty.c.orig	Thu May 11 03:55:14 2000
+++ ./imontty/imontty.c	Fri Jun 29 11:14:07 2001
@@ -12,6 +12,14 @@
 #include <linux/isdn.h>
 #include "imontty.h"
 
+/* Ugly hack, but the isdn header now defines it in kernel space. */
+#ifndef ISDN_MAX_CHANNELS
+#  define ISDN_MAX_CHANNELS 64
+#endif
+#ifndef ISDN_MAX_DRIVERS
+#  define ISDN_MAX_DRIVERS 32
+#endif
+
 struct phone_entry {
    struct phone_entry *next;
    char phone[30];
--- ./imon/imon.c.orig	Sat May 17 05:23:35 1997
+++ ./imon/imon.c	Fri Jun 29 11:14:21 2001
@@ -56,6 +56,11 @@
 
 #include <linux/isdn.h>
 
+/* Ugly hack, but the isdn header now defines it in kernel space. */
+#ifndef ISDN_MAX_CHANNELS
+#  define ISDN_MAX_CHANNELS 64
+#endif
+
 #define KEY_Q                81
 #define KEY_q               113
 
--- ./xmonisdn/Net.c.orig       Tue Aug  3 07:00:20 1999
+++ ./xmonisdn/Net.c    Fri Jun 29 12:03:25 2001
@@ -49,6 +49,11 @@
 #include <X11/Xmu/Drawing.h>
 #include <X11/extensions/shape.h>
 
+/* Ugly hack, but the isdn header now defines it in kernel space. */
+#ifndef ISDN_MAX_CHANNELS
+#  define ISDN_MAX_CHANNELS 64
+#endif
+
 /* Only allow calling scripts when setuid root if the -r option is given */
 extern int allow_setuid;
 /*
